null_safety/calculate_lix/text/lorem-ipsum.txt

```diff
@@ -0,0 +1,19 @@
+Lorem. ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
```

Comment:
Would be cool to also add a more readable file, for comparison. Some Dr. Seuss, or a kid song?

Reply:
Careful of the copyright!

Maybe Beowulf? 😜